### DISCUSSION TITLE
Bug? metrics gets cut off

### DISCUSSION THREAD

**Discussion by ProFireDev** [lowlighter/metrics#795] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/34112557/150399628-2ec4f71f-951b-492d-ae2e-5a837af18d9f.png)

^the bottom text is pushed off the svg on my profile. My config and the svg are attached below in the zip
[ProFireDev-main.zip](https://github.com/lowlighter/metrics/files/7907673/ProFireDev-main.zip)

**Comment by lowlighter** [comment, marked as answer]
(was addressed in #796)
